config/core/LocalSettings.php

```diff
@@ -1076,71 +1173,71 @@
-$wgNamespaceContentModels[NS_USER_TALK]           = CONTENT_MODEL_FLOW_BOARD;
-$wgNamespaceContentModels[NS_TALK]                = CONTENT_MODEL_FLOW_BOARD;
-$wgNamespaceContentModels[NS_HELP_TALK]           = CONTENT_MODEL_FLOW_BOARD;
-$wgNamespaceContentModels[NS_FILE_TALK]           = CONTENT_MODEL_FLOW_BOARD;
-$wgNamespaceContentModels[NS_CATEGORY_TALK]       = CONTENT_MODEL_FLOW_BOARD;
-$wgNamespaceContentModels[NS_MEDIAWIKI_TALK]      = CONTENT_MODEL_FLOW_BOARD;
-$wgNamespaceContentModels[NS_TEMPLATE_TALK]       = CONTENT_MODEL_FLOW_BOARD;
-$wgNamespaceContentModels[SMW_NS_FORM_TALK]       = CONTENT_MODEL_FLOW_BOARD;
-$wgNamespaceContentModels[SMW_NS_PROPERTY_TALK]   = CONTENT_MODEL_FLOW_BOARD;
-$wgNamespaceContentModels[SMW_NS_CONCEPT_TALK]    = CONTENT_MODEL_FLOW_BOARD;
+#
+# Extension:Flow
+#
+# Note: Flow removed due to being unable to search discussions. While the
+# improved interface is great, it's useless if we can't search our old content.
+# See issues #272.
+#
+// require_once $egExtensionLoader->registerLegacyExtension(
+// 	'Flow',
+// 	'https://gerrit.wikimedia.org/r/mediawiki/extensions/Flow.git',
+// 	'REL1_25'
+// );
+
+// // only allow sysops to create new flow boards
+// $wgGroupPermissions['sysop']['flow-create-board'] = true;
 
-// Connect Flow to Parsoid
-$wgFlowParsoidURL = 'http://127.0.0.1:8000';
-$wgFlowParsoidPrefix = $wikiId;
+// // store posts as html using Parsoid
+// $wgFlowContentFormat = 'html';
 
+// // use VE
+// $wgFlowEditorList = array( 'visualeditor', 'none' );
 
+// // Define which namespaces will use Flow
+// $wgNamespaceContentModels[NS_PROJECT_TALK]        = CONTENT_MODEL_FLOW_BOARD;
+// $wgNamespaceContentModels[NS_USER_TALK]           = CONTENT_MODEL_FLOW_BOARD;
+// $wgNamespaceContentModels[NS_TALK]                = CONTENT_MODEL_FLOW_BOARD;
+// $wgNamespaceContentModels[NS_HELP_TALK]           = CONTENT_MODEL_FLOW_BOARD;
+// $wgNamespaceContentModels[NS_FILE_TALK]           = CONTENT_MODEL_FLOW_BOARD;
+// $wgNamespaceContentModels[NS_CATEGORY_TALK]       = CONTENT_MODEL_FLOW_BOARD;
+// $wgNamespaceContentModels[NS_MEDIAWIKI_TALK]      = CONTENT_MODEL_FLOW_BOARD;
+// $wgNamespaceContentModels[NS_TEMPLATE_TALK]       = CONTENT_MODEL_FLOW_BOARD;
+// $wgNamespaceContentModels[SMW_NS_FORM_TALK]       = CONTENT_MODEL_FLOW_BOARD; // MW throws error: SMW_NS_FORM_TALK not a constant
+// $wgNamespaceContentModels[SMW_NS_PROPERTY_TALK]   = CONTENT_MODEL_FLOW_BOARD;
+// $wgNamespaceContentModels[SMW_NS_CONCEPT_TALK]    = CONTENT_MODEL_FLOW_BOARD;
 
+// // Connect Flow to Parsoid
+// $wgFlowParsoidURL = 'http://127.0.0.1:8000';
+// $wgFlowParsoidPrefix = $wikiId;
 
 
 
 
 
 
 /**
- *  8) LOAD OVERRIDES
+ *  9) LOAD OVERRIDES
  *
  *
  *
  *
  **/
-if ( file_exists( "$m_htdocs/wikis/$wikiId/config/overrides.php" ) ) {
-	require_once "$m_htdocs/wikis/$wikiId/config/overrides.php";
+if ( file_exists( "$m_config/local/postLocalSettings_allWikis.php" ) ) {
```

Comment:
Off the top of my head I don't think this is generated anywhere, and it doesn't need to be. It's for customization, if desired (hence `if( file_exists( ... ) )`). 

Anything that is specified in `LocalSettings.php` would override anything specified in `preLocalSettings_allWikis.php`. Things not specified in `LocalSettings.php` that are specified in `preLocalSettings_allWikis.php` would not be overridden, however. So `preLocalSettings_allWikis.php` may be useful in some cases.